frontend/src/components/profile/Edit.vue

```diff
@@ -35,22 +35,30 @@
         <ul class="list-group">
           <li class="list-group-item">
             <span v-if="orcid_url">
-              <a :href="orcid_url"><span class="text-gray fab fa-orcid"></span> {{ orcid_url }}</a>
+              <a :href="orcid_url">
+                <span class="text-gray fab fa-orcid"></span> {{ orcid_url }}
+              </a>
+              <a title="Edit connection" :href="socialConnectionsURL">
+                <i class="float-right fas fa-edit"></i>
+              </a>
             </span>
             <span v-else>
-              <a title="orcid" href="/accounts/orcid/login/?process=connect"
-                ><span class="text-gray fab fa-orcid"></span> Connect your ORCID account</a
-              >
+              <a title="github" :href="socialConnectionsURL">
```

Comment:
We could either get rid of the  `a title` here and elsewhere in this text or add more semantic info to them like "sign in with orcid", "sign in with github" etc

Reply:
cleaned up the markup around here + some other consistency changes